mlflow/tracking/client.py

```diff
@@ -1316,6 +1882,50 @@ def update_model_version(self, name, version, description=None):
         :param description: New description.
 
         :return: A single :py:class:`mlflow.entities.model_registry.ModelVersion` object.
+
+        .. code-block:: python
+            :caption: Example
+
+            import mlflow.sklearn
+            from mlflow.tracking import MlflowClient
+            from sklearn.ensemble import RandomForestRegressor
+
+            def print_model_version_info(mv):
+                print("Name: {}".format(mv.name))
+                print("Version: {}".format(mv.version))
+                print("Description: {}".format(mv.description))
+
+            mlflow.set_tracking_uri("sqlite:///mlruns.db")
+            params = {"n_estimators": 3, "random_state": 42}
+            name = "RandomForestRegression"
+            rfr = RandomForestRegressor(**params)
+
+            # Log MLflow entities
+            with mlflow.start_run() as run:
+                mlflow.log_params(params)
+                mlflow.sklearn.log_model(rfr, artifact_path="sklearn-model")
+
+            # Create a new version of the rfr model under the registered model name
+            model_uri = "runs:/{}/sklearn-model".format(run.info.run_id)
+            mv = client.create_model_version(name, model_uri, run.info.run_id)
```

Comment:
`client` is not defined in this code snippet.

Reply:
I think `create_registered_model` needs to be called before `create_model_version`.

Reply:
Yeah, missed from the cut-n-paste from [tests](https://github.com/dmatrix/mlflow-tests/tree/master/py/mlflow/apis/client)
```
# Register model name in the model registry
client = MlflowClient()
client.create_registered_model(name)
```